lib/WP_Auth0_Options.php

```diff
@@ -129,92 +134,124 @@ public function get_logout_url() {
 		return add_query_arg( 'action', 'logout', site_url( 'wp-login.php', 'login' ) );
 	}
 
+	/**
+	 * Get as a string-separated string and parse to array
+	 *
+	 * @return array
+	 */
+	public function get_lock_connections() {
+		$connections = $this->get( 'lock_connections' );
+		$connections = empty( $connections ) ? array() : explode( ',', $connections );
+		return array_map( 'trim', $connections );
```

Comment:
is the trim needed because it's a user input or because you don't trust the source? (i don't know where it comes from)

Reply:
User input